### ISSUE TITLE
Why did you move downloads? 

### ISSUE THREAD

**Issue by qudo-code** [brave/brave-browser#29828]
## Description 
Why did downloads move to an additional menu I have to now click, then open a folder, and THEN drag my download to what I want. Why did brave force 3 more clicks on me and opening up file explorer for absolutely no reason, I am annoyed. What is this, Safari? 

![Screenshot 2023-04-18 at 11 27 03 AM](https://user-images.githubusercontent.com/59782726/232869739-f33a38dd-2aac-4558-8fdc-9f083ce6c6f2.png)

## Steps to Reproduce
Use brave, download something, be sad. 

I hate this,
sincerely,
qudo

**Comment by LCDraws** [comment]
what i REALLY FREAKING HATE is that the panel auto opens after adding something to download, KILLING my keyboard inputs, mouse clicks, etc... SO DAMN ANNOYING. Add an option to never auto open that panel or roll it back, no one asked for this.

**Comment by qudo-code** [comment]
Yeah, it take focus away. Not only made downloads extremely inconvenient but broke so many other flows as well. I now have to reach for my mouse because of Brave when I wouldn't have otherwise.

**Comment by qudo-code** [comment]
Day 5 of me still hating brave because of this dumb UX decision. I download so many things and this has literally doubled the time of my normal workflow.

**Comment by qudo-code** [comment]
+1

**Comment by StrangePeanut** [comment]
This change made me hop into `brave:flags` for the first time ever just to disable it. I think they were going for a downloading experience similar to that on Edge, but it is clearly not ready and should never have made it to stable in its current state imo.

**Comment by LCDraws** [comment]
> This change made me hop into `brave:flags` for the first time ever just to disable it. I think they were going for a downloading experience similar to that on Edge, but it is clearly not ready and should never have made it to stable in its current state imo.

Oh you saved me, was super easy to disable. Thanks :)

**Comment by adler-amorette** [comment]
Setting `--disable-features=DownloadBubble` used to work, now it doesn't.

One thing not mentioned here is that because the download bubble pops up by itself and has a delete button, it's very possible to accidentally delete a file when you meant to click on a web page.

The download bar must be brought back.
Current UX is horrid.

**Comment by OcBuddy** [comment]
> Setting `--disable-features=DownloadBubble` used to work, now it doesn't.
> 
> One thing not mentioned here is that because the download bubble pops up by itself and has a delete button, it's very possible to accidentally delete a file when you meant to click on a web page.
> 
> The download bar must be brought back. Current UX is horrid.

Did they change download code?
Because since 2 days or so my download filder is not remembered and it always wants to download to the configured download folder. Previously when 'ask' was checked it would remember the previously used folder.
